public/_posts/_articles/marco-worms/yearn-governance/es.md

```diff
@@ -0,0 +1,128 @@
+---
+title:  "Explicación de la gobernanza de Yearn: propuestas, token $YFI y ejecución"
+image:
+  src: ./image1.jpg
+  width: 900
+  height: 482
+date: '2022-04-19'
+author: Marco_Worms
+translator: RestlessMik3
+---
+
+![](./image1.jpg?w=900&h=482)
+
+Cualquiera puede discutir o proponer ideas en Yearn's [Discord](https://discord.com/invite/6PNv2nF), [Twitter](https://twitter.com/iearnfinance), [Reddit](https://www.reddit.com/r/yearn_finance), [Pool of Ideas](https://yearnfinance.notion.site/yearnfinance/Pool-of-Ideas-d75383ade9154d8bb6163388c6c2b39b), [Telegram](https://t.me/yearnfinance/), y [Foro de Gobernanza](https://gov.yearn.finance/). A medida que maduran las propuestas, se convierte en un voto estructurado en el Foro de Gobernanza donde el [token YFI](https://www.coingecko.com/en/coins/yearn-finance) se usa como poder de voto.
```

Comment:
AS-IS: [Pool of Ideas]

TO-BE: [Listado de Ideas]

Reply:
Done